templates-global/hero.php

```diff
@@ -22,17 +23,43 @@
 	$hero_highlight = '';
 }
 
+/**
+ * The text color of the hero title is usually black. However, when 'none' is
+ * chosen, we allow the user to choose between black or white text to best fit
+ * with the image.
+ */
+if ( 'white' == $title_color && 'none' == $hero_highlight || empty( $hero_highlight ) ) {
+	$title_color_class = 'text-white';
+} else {
+	$title_color_class = '';
+}
+
```

Comment:
'black' option is missing here but was added to the ACF json file. 
`elseif ( 'black' == $title_color && 'none' == $hero_highlight || empty( $hero_highlight ) )`

Also, I would remove the other side of the condition `('none' == $hero_highlight || empty( $hero_highlight ))`, since the text color option won't show unless the $hero_highlight is none anyway.